renku_notebooks/util/kubernetes_.py

```diff
@@ -299,3 +300,77 @@ def _secret_exists(name, namespace):
     except client.rest.ApiException:
         pass
     return False
+
+
+def create_pvc(
+    name,
+    username,
+    git_namespace,
+    project_id,
+    commit_sha,
+    git_host,
+    storage_size,
+    storage_class="default",
+):
+    """Create a PVC."""
+
+    # check if we already have this PVC
+    pvc = _get_pvc(username, project_id, commit_sha)
+
+    if pvc:
```

Comment:
Should we also check for the size and adjust it if the user required more GB?

Reply:
Right, I also thought that might be confusing - it should be possible to resize the volumes afaik, I will try.